rosbag2_transport/src/rosbag2_transport/rosbag2_transport_python.cpp

```diff
@@ -46,22 +47,25 @@ rosbag2_transport_record(PyObject * Py_UNUSED(self), PyObject * args, PyObject *
   bool all = false;
   bool no_discovery = false;
   uint64_t polling_interval_ms = 100;
+  uint64_t max_bagfile_size = UINT64_MAX;
```

Comment:
This does not match the doc:
https://docs.python.org/3/c-api/arg.html#numbers

It should be an `unsigned long long` and not an `uint64_t`. The current code is also wrong on 32-bits arch. See `stdint.h`:

```c++
#if __WORDSIZE == 64
typedef unsigned long int   uint64_t;
#else
__extension__
typedef unsigned long long int  uint64_t;
#endif
```